test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -311,6 +323,36 @@ def test_dependency_check_neuron(neuron, ec2_connection):
     _run_dependency_check_test(neuron, ec2_connection, "neuron")
 
 
+@pytest.mark.model("N/A")
+def test_dataclasses_check(image):
+    """
+    Ensure there is no dataclasses pip package is installed for python 3.7 and above version.
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    pip_package = "dataclasses"
+
+    container_name = get_container_name("dataclasses-check", image)
+
+    python_version = re.findall(r'py(\d\.?\d?)', image)[0]
+    python_version = int(python_version)
+
+    if python_version >= 37:
+        start_container(container_name, image, ctx)
+        output = run_cmd_on_container(
+            container_name, ctx, f"pip show {pip_package}", warn=True)
+
+        if output.return_code == 0:
+            pytest.fail(
+                f"{pip_package} package exists in the DLC image {image} that has py{python_version} version which is greater than py36 version")
+        else:
+            LOGGER.info(
+                f"{pip_package} package does not exists in the DLC image {image}")
+    else:
```

Comment:
nit: you can probably add a pytest.skip here so that the skip reason shows up in the summary as "Skipped" instead of "Passed"